server/src/main.rs

```diff
@@ -0,0 +1,55 @@
+use clap::{App, Arg};
+use shadow_rs::shadow;
+
+pub mod management_api {
+    tonic::include_proto!("management");
+}
+pub mod attestation_api {
+    tonic::include_proto!("attestation");
+}
+
+#[macro_use]
+extern crate log;
+shadow!(build);
+
+mod attestation;
+mod management;
+mod server;
+mod users;
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+
+    let version = format!(
+        "\nv{}\ncommit: {}\nbuildtime: {}",
+        build::PKG_VERSION,
+        build::COMMIT_HASH,
+        build::BUILD_TIME
+    );
+
+    let matches = App::new("attestation-server")
+        .version(version.as_str())
+        .long_version(version.as_str())
+        .author("Confidential-Containers Team")
+        .arg(
+            Arg::with_name("listen")
+                .short("l")
+                .long("listen")
+                .value_name("sockaddr")
+                .help("Socket that the server will listen on to accept attestation & configuration requests.")
+                .takes_value(true),
+        )
+        .get_matches();
+
+    let sockaddr = match matches.is_present("listen") {
```

Comment:
Maybe try something along the lines of `let sockaddr = matches.value_of("listen").unwrap_or("default".to_string());`